crates/pallet-feeds/src/mock.rs

```diff
@@ -56,16 +59,35 @@ parameter_types! {
     pub const MaxFeeds: u32 = 1;
 }
 
+#[derive(Debug, Clone, Encode, Decode, TypeInfo, Eq, PartialEq)]
+pub enum FeedProcessorKind {
+    FullObject(Option<Vec<u8>>),
+    Content(Option<Vec<u8>>),
+    ContentEnum(Option<Vec<u8>>),
+}
```

Comment:
Why do we need feed processor kind contain an optional vector of bytes? This seems really weird.

Shouldn't that option be part of initialization, if at all? My guess it might be related to the API design somehow, but feels too complex.

Reply:
These are mocks feed processors for the tests. Wont be part of the runtime.

Reply:
The optionals there are injecting the key to override.

Reply:
Why can't those things be initialization arguments then? I just don't see why feed processor kind should contain any additional contents in it.

Reply:
yeah. that works too ;)